draft-ietf-quic-multipath.md

```diff
@@ -379,25 +379,26 @@ should not cause linkability issue.
 
 # Path Management {#path-management}
 
-After completing the handshake, endpoints have agreed to enable
-multipath support. They can start using multiple paths when both endpoints
-have issued available connection IDs for at least one unused Path ID.
-This document does not discuss when a client decides to initiate a new path.
+After completing the handshake indicating
+multipath support, endpoints can start using multiple paths when both endpoints
+have issued available connection IDs for at least one unused, common Path ID,
+as the same Path ID is used in both directions.
+
```

Comment:
Not sure that this rewrite is better than the original. You could just strike out the last line, "This document does not discuss when a client decides to initiate a new path."

Reply:
The initial text is not correct. 

"After completing the handshake, endpoints have agreed to enable
multipath support." -> only if both ends indicated multipath support (but I don't think we need to repeat this here.

I think this happened because this used to be use sentence we split it up (wrongly) at some point.

Please see another proposal to split this into two sentence above.

Reply:
New proposal works.